src/Microsoft.Win32.Registry/tests/RegistryTestsBase.cs

```diff
@@ -24,7 +24,13 @@ protected RegistryTestsBase()
             RemoveKeyIfExists(TestRegistryKeyName);
 
             // Then create the key.
+#if net46
```

Comment:
Can we do this with some mechanism other than ifdefs?  We've been trying hard to avoid them, and thus far have been fairly successful.  Is there a runtime check we could do?  Could we split this into a net46 test and a corefx test? Etc.